### PR TITLE
Fixes #24

### LINKED ISSUE #24: Add get_case tests
Add tests that check is the case of word has been determined properly.

### PR DESCRIPTION
**Pull request by savermyas**
Added a couple of tests for case definition

### CODE REVIEW

File: test/testAnalyser.py [hunk 1]
```diff
@@ -31,9 +31,12 @@ def test_get_structured_sentence(self):
                           'сказал', ' ', 'Отец', '!'])
 
     def test_get_case(self):
-        name = Name("", "", "", "")
-        name.preposition = "на"
-        name.text = "Коле"
+        name = Name("на", "Кост", "", "е")
+        self.assertEqual(text.analyser.get_case(name), 2)
```
Comment (krokopluh) [inline test/testAnalyser.py]: This must be a third case, i suppose.

File: test/testAnalyser.py [hunk 2]
```diff
@@ -31,9 +31,12 @@ def test_get_structured_sentence(self):
...
+        name = Name("", "Виктор", "", "")
+        self.assertEqual(text.analyser.get_case(name), 0)
+        name = Name("", "Дим", "он", "ом")
+        self.assertEqual(text.analyser.get_case(name), 5)
```
Comment (krokopluh) [inline test/testAnalyser.py]: This must be a 4th case

**Review by krokopluh — CHANGES_REQUESTED** [review]
Your tests will fail all the time

File: test/testAnalyser.py (hunk 1 above)
Comment (savermyas) [inline test/testAnalyser.py]: it is a datelny padezh

**Comment by savermyas** [comment]
updated

**Pull request merged**